docs/docs/migration/v0.22.2.md

```diff
@@ -0,0 +1,21 @@
+---
+sidebar_position: 0
+title: v0.22.2
+description: For chains that were scaffolded with Ignite CLI versions lower than v0.22.2, changes are required to use Ignite CLI v0.22.2. 
+---
+
+# Upgrading a blockchain to use Ignite CLI v0.22.2
+
+Ignite CLI v0.22.2 changed the GitHub username from "ignite-hq" to "ignite".
+
+1. Open your `go.mod` and change the Ignite CLI line with `github.com/ignite/cli v0.22.2`
+
+2. In your `go.sum` file change `github.com/ignite-hq/cli` to `github.com/ignite/cli`
```

Comment:
I think it is better just running `go mod tidy` here, it'll auto adjust the go.sum file.